go.mod

```diff
@@ -64,13 +67,16 @@ require (
 	gopkg.in/macaroon.v2 v2.0.0
 )
 
+require github.com/charmbracelet/x/ansi v0.1.4 // indirect
+
 require (
 	github.com/Azure/go-ansiterm v0.0.0-20230124172434-306776ec8161 // indirect
 	github.com/Microsoft/go-winio v0.6.1 // indirect
 	github.com/NebulousLabs/fastrand v0.0.0-20181203155948-6fb6489aac4e // indirect
 	github.com/Nvveen/Gotty v0.0.0-20120604004816-cd527374f1e5 // indirect
 	github.com/aead/chacha20 v0.0.0-20180709150244-8b13a72661da // indirect
 	github.com/aead/siphash v1.0.1 // indirect
+	github.com/aymanbagabas/go-osc52/v2 v2.0.1 // indirect
```

Comment:
Hmm, that's quite a few new dependencies being pulled in just for the coloring... Maybe we should for now leave the `lipgloss` part out and just do everything else? The structured log with attributes and context-based tracing-values plus all the console/file handling already gives us a ton, wouldn't want to see this PR being blocked by safety concerns. What do you think?
Or maybe we can put the coloring stuff behind a not-yet enabled build tag, so at least at build time these dependencies aren't pulled in (yet), until we can take a closer look at them?

Reply:
(removed commit)